github/orgs.go

```diff
@@ -70,6 +70,11 @@ type Organization struct {
 	ReposURL         *string `json:"repos_url,omitempty"`
 }
 
+type OrganizationInstallations struct {
+	TotalCount    *int           `json:"total_count,omitempty"`
+	Installations []Installation `json:"installations,omitempty"`
```

Comment:
Please change this to `[]*Installation`.

Reply:
Done